tc-server/src/main/java/com/tc/l2/logging/TCLogbackLogging.java

```diff
@@ -122,13 +128,19 @@ public static void redirectLogging(File logDirFile) {
     } else {
       disableBufferingAppender(null);
     }
+
+    Appender<ILoggingEvent> bootstrap = root.getAppender("TC_BASE");
+    if (bootstrap != null) {
+      root.detachAppender(bootstrap);
+      try { bootstrap.stop(); } catch (Exception ignore) {}
+    }
```

Comment:
We still want console output going to the output stream provided to bootstrap logging.